nuget/FSharp.Formatting.nuspec

```diff
@@ -20,15 +20,15 @@
     <!-- Automatically referenced by VS -->
     <!-- When including additional .dll files here make sure you update "generate.fsx" as well 
          (it will trigger a CI build failure when forgotten so not a problem) -->
-    <file src="..\bin\CSharpFormat.*" target="lib/net40" />
-    <file src="..\bin\FSharp.CodeFormat.*" target="lib/net40" />
-    <file src="..\bin\FSharp.Literate.*" target="lib/net40" />
-    <file src="..\bin\FSharp.Markdown.*" target="lib/net40" />
-    <file src="..\bin\FSharp.Formatting.Common.*" target="lib/net40" />
-    <file src="..\bin\FSharp.Formatting.Razor.*" target="lib/net40" />
-    <file src="..\bin\FSharp.MetadataFormat.*" target="lib/net40" />
-    <file src="..\bin\RazorEngine.*" target="lib/net40" />
-    <file src="..\bin\System.Web.Razor.*" target="lib/net40" />
+    <file src="..\bin\net461\CSharpFormat.*" target="lib/net461" />
```

Comment:
Then we need to update `FSFormatting.fsx` as well. This is a breaking change but fine with me (for historic usage with relative paths)

Reply:
FSharp.Formatting.fsx has been updated with `#I "lib/net461"`

Reply:
And the projects where already compiled with a target framework net461, but the nuget package was still net45... which could cause some problems.